charts/internal/seed-controlplane/charts/cloud-provider-equinix-metal/templates/vpa.yaml

```diff
@@ -10,3 +10,10 @@ spec:
     name: cloud-controller-manager
   updatePolicy:
     updateMode: Auto
+  resourcePolicy:
+    containerPolicies:
+    - containerName: cloud-provider-equinix-metal
+      maxAllowed:
+        cpu: 4
+        memory: 10G
+      controlledValues: RequestsOnly
```

Comment:
I don't think we should switch to `RequestsOnly`

Reply:
Done